nautobot/extras/migrations/0117_task_queue_to_job_queue_migration.py

```diff
@@ -11,10 +12,17 @@ def migrate_task_queues_to_job_queues(apps, schema):
     ScheduledJob = apps.get_model("extras", "ScheduledJob")
     JobQueue = apps.get_model("extras", "JobQueue")
 
+    default_job_queue, _ = JobQueue.object.get_or_create(
```

Comment:
```suggestion
    default_job_queue, _ = JobQueue.objects.get_or_create(
```

Reply:
My vscode syntax highlighting is all messed up for some reason thank you for catching it